charts/v1.6.0/azurefile-csi-driver/templates/csi-snapshot-controller.yaml

```diff
@@ -22,6 +22,11 @@ spec:
 {{- with .Values.controller.tolerations }}
       tolerations:
 {{ toYaml . | indent 8 }}
+{{- end }}
+{{- with .Values.controller.nodeAffinity }}
+      affinity:
+        nodeAffinity:
+{{ toYaml . | indent 10 }}
```

Comment:
Maybe we could change `.Values.controller.nodeAffinity` to `.Values.snapshot.snapshotController.affinity`.

we may want to split snapshot controller affinity and azurefile controller :thinking:  ?